version.props

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectReunionVersionMajor>1</ProjectReunionVersionMajor>
+    <ProjectReunionVersionMinor>0</ProjectReunionVersionMinor>
```

Comment:
it's hard to go backwards especially on nuget.org. Recommend starting version at 0.1 while we're baking stuff.

Reply:
thank you, changing now.